### DISCUSSION TITLE
Custom help command.

### DISCUSSION THREAD

**Discussion by cmachaca** [daniele77/cli#142] [Q&A]
Hello.

Is there a way to disable the default commands? Can you also add custom loggers?

**Comment by daniele77** [comment]
Hi @cmachaca 
With "default commands" do you mean "exit" and "help" command?
And what do you mean by "custom loggers"? Do you need to trace the library internal path?

**Reply by cmachaca** [reply]
The idea is to allow custom exit and help commands and thus use a single ExitAction method. The help command cannot display the commands in a custom way.

**Comment by daniele77** [comment]
I'm afraid currently there is no way to customize exit and help commands.
In the help command would you like to change the formatting? You can already specify the content in the `Menu::Insert` method (where you specify the description of the command and its parameters).
